### PR TITLE
Move to standalone TSC in CI configuration

### PR DESCRIPTION
**Pull request by neonichu**
Supersedes #305

### CODE REVIEW

**Comment by benlangmuir** [comment]
https://github.com/apple/swift-package-manager/pull/2991
@swift-ci please test

**Review by benlangmuir — APPROVED** [review]
Approved once TSC gets PR testing that includes sourcekit-lsp.

**Comment by neonichu** [comment]
TSC has the extended PR testing now, running CI on https://github.com/apple/swift-package-manager/pull/2991 once more, then this should be good to merge.

**Pull request merged**
